robotbase/src/main/java/frc/robot/Constants.java

```diff
@@ -222,6 +225,12 @@ public static final class LATERATOR {
         .maxAcceleration(0)
         .maxVelocity(0);
 
+    public static final double GEAR_RATIO = 3 / 1;
+    public static final Distance MOTOR_PULLEY_PITCH_DIAMETER =
+      Units.Millimeters.of(46.188);
+    public static final Distance MOTOR_PULLEY_PITCH_CIRCUMFERENCE =
+      MOTOR_PULLEY_PITCH_DIAMETER.times(Math.PI);
```

Comment:
Can these be renamed to OUTPUT_PULLEY instead of MOTOR_PULLEY?

Also, just want to confirm where the 46.188 came from to make sure it's right.

Reply:
Zach gave it to me